components/NetworkInformation/network-information.mdx

```diff
@@ -0,0 +1,17 @@
+import { Callout } from "nextra/components";
+
+| Name            | Value                                  |
+| --------------- | -------------------------------------- |
+| Network Name    | Superseed Sepolia Testnet              |
+| Description     | The public testnet for Superseed.      |
+| RPC Endpoint    | https://sepolia.superseed.xyz          |
+| Chain ID        | 53302                                  |
+| Currency Symbol | ETH                                    |
+| Block Explorer  | https://sepolia-explorer.superseed.xyz |
+
+---
+
+<Callout emoji="⚠️" type="warning">
+  *Our RPC endpoints are Rate limited and not recommended for production
+  systems.*
+</Callout>
```

Comment:
I'd also move this callout to the Network Information page so that we don't display it in Using Superseed

Reply:
Ah ok, no problem!